bucket/foobar2000.json

```diff
@@ -24,9 +18,17 @@
     "installer": {
         "script": [
             "Remove-Item \"$dir\\`$*\", \"$dir\\uninstall.exe\" -Force -Recurse",
-            "New-Item \"$dir\\user_profiles_enabled\" -Force | Out-Null"
+            "New-Item \"$dir\\portable_mode_enabled\" -Force | Out-Null"
         ]
     },
+    "post_install": [
+        "if(Test-Path \"$persist_dir\\playlists\") {",
+        "    Rename-Item -Path \"$persist_dir\\playlists\" -NewName playlists-v1.4 -Force",
+        "    Get-ChildItem -Exclude \"profile\" -Path \"$persist_dir\" |",
+        "        Move-Item -Destination \"$persist_dir\\profile\" -Force",
+        "}"
+    ],
+    "persist": "profile",
```

Comment:
require data migration from `~\AppData\Roaming\foobar2000`

Reply:
Will look at it

Reply:
Added